app/protocols/theme-handler.js

```diff
@@ -0,0 +1,54 @@
+import path from 'path'
+import { fileURLToPath } from 'url'
+import fs from 'fs'
+import mime from 'mime-types'
+import { Readable } from 'stream'
+
+const __dirname = fileURLToPath(new URL('./', import.meta.url))
+
+export async function createThemeHandler () {
+  return {
+    handler: async function protocolHandler ({ url }, sendResponse) {
+      const parsedUrl = new URL(url)
+      let filePath
+
+      if (parsedUrl.hostname === 'theme') {
```

Comment:
We should have a check for `pathname === '/vars.css'` and generate it dynamically the way we do in the current handler here: https://github.com/AgregoreWeb/agregore-browser/blob/master/app/protocols/browser-protocol.js#L76

That's also where the base16 vars should get defined.